dmd_IN.m

```diff
@@ -0,0 +1,49 @@
+function IN=dmd_IN(N,b)
+%% 
+% dmd_IN.m calculates IN, which is used in the calculation of D1
+%
+% Parameters:
+%   N: dimension of dirac mixture approximation
+%   b: upper bound for integral
+%
+% Return values:
+%   IN: value used in the calculation of D1
+%
+% History:
+% * 2018/01/04 Dantong Wang
+% * 2018/01/25 Dantong Wang
+
+%% I1 and I2
+k=floor(N/2);
+l=N-k*2;
+I1=1/2*(b*(1+b^2)^(1/2)-asinh(b));
+I2=1/2*(b^2-log(1+b^2));
+
+%% calculate IN
+%if N<02
+if N==1
+    IN=I1
+else if N==2 
```

Comment:
Shouldn't this be an "elseif"?
I think one could restructure this and make it more readable...